app/controllers/venues_controller.rb

```diff
@@ -2,6 +2,6 @@ class VenuesController < ApplicationController
 
   def show
     @venue = Venue.find_by(id: params[:id])
-    @events = Event.active_events(@venue)
+    @events = @venue.open_events
```

Comment:
I don't know that this works. We're calling an event method on the a venue object. The purpose for the original code on the left was to specify active events for a specific venue.